cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/LowestAddressJoinDecider.scala

```diff
@@ -74,7 +74,7 @@ class LowestAddressJoinDecider(system: ActorSystem, settings: ClusterBootstrapSe
                 lowestAddress.map(contactPointString).getOrElse(""),
                 info.contactPoints.map(contactPointString).mkString(", ")
               )
-            else
+            else if (log.isWarningEnabled)
```

Comment:
In the previous code, this logging was only displaying if `newClusterEnabled` was false. That's not the case anymore.

Reply:
Yeah, `scalafmt` screwed me here. I'll just add curly braces.

Reply:
See logging below:

```
Exceeded stable margins without locating seed-nodes, however this node {} is configured with 
new-cluster-enabled=off, thus NOT joining self. Expecting existing cluster or node [{}] 
(out of [{}]) to perform the self-join and initiate the cluster.
```

The full log says, that `new-cluster-enabled=off`.

Reply:
Fixed it up

Reply:
Thanks for the quick reaction. 👍